packages/ui/src/components/notifications/Notifications.svelte

```diff
@@ -0,0 +1,61 @@
+<script lang="ts">
+  import { setContext } from 'svelte'
+
+  import Notification from './Notification.svelte'
+  import store, { notificationsContextKey } from './store'
+  import { NotificationPosition } from './NotificationPosition'
+
+  const getClass = (position: NotificationPosition) => {
```

Comment:
You could use record with positions.
```suggestion
const toPosition = {
  [NotificationPosition.BottomLeft]: 'bottom-left',
  [NotificationPosition.BottomRight]: 'bottom-right',
  [NotificationPosition.TopLeft: 'top-left',
  [NotificationPosition.TopRight]: 'top-right'
   }
```

Reply:
Done.